<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="5" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-12T17:03:12 IST" name="Default suite" finished-at="2022-11-12T17:03:12 IST" duration-ms="433">
    <groups>
    </groups>
    <test started-at="2022-11-12T17:03:12 IST" name="Default test" finished-at="2022-11-12T17:03:12 IST" duration-ms="433">
      <class name="rahulshettypractice.End2EndCart">
        <test-method signature="testFour()[pri:-2, instance:rahulshettypractice.End2EndCart@7a35b0f5]" started-at="2022-11-12T17:03:12 IST" name="testFour" finished-at="2022-11-12T17:03:12 IST" duration-ms="28" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFour -->
        <test-method signature="testTwo()[pri:-1, instance:rahulshettypractice.End2EndCart@7a35b0f5]" started-at="2022-11-12T17:03:12 IST" name="testTwo" finished-at="2022-11-12T17:03:12 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTwo -->
        <test-method signature="testOne()[pri:0, instance:rahulshettypractice.End2EndCart@7a35b0f5]" started-at="2022-11-12T17:03:12 IST" name="testOne" finished-at="2022-11-12T17:03:12 IST" duration-ms="14" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOne -->
        <test-method signature="testP()[pri:0, instance:rahulshettypractice.End2EndCart@7a35b0f5]" started-at="2022-11-12T17:03:12 IST" name="testP" finished-at="2022-11-12T17:03:12 IST" duration-ms="12" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at rahulshettypractice.End2EndCart.testP(End2EndCart.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testP -->
      </class> <!-- rahulshettypractice.End2EndCart -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
